### PR TITLE
Telecommunications network monitoring console improvements

### PR DESCRIPTION
**Pull request by Kurfursten**
The telecommunications network monitoring console is sort of the redheaded stepchild of telecoms computers. Your server monitor and message monitor are useful chat logs, the traffic controller is used for scripting, but the network monitor is only good for checking links which is a rarer form of sabotage. Similarly, the most common cause of telecoms blackout is the random event, but a mechanic who diagnoses this problem can't do anything to fix it but wait it out. Now when viewing an individual machine, it will display the integrity of that machine and, if it has been affected by an EMP (usually from the comms blackout event), you can signal boost the machine to get it working again immediately. The signal boost button will not appear if the machine is not EMPED.

untitled

🆑 
* rscadd: The telecommunications network monitoring console can now boost the signal of telecomms machines affected by ionospheric interference, rebooting them immediately. This will cause increased temperature and power expense for five minutes.
* rscadd: The network monitor will also remotely display machine integrity.

### CODE REVIEW

**Comment by gbasood** [comment]
not sure about the boosting comms thing
not sure if i should care enough about it to say it's bad, that is
the event shouldn't really be bypassable in this way at least

**Comment by Kurfursten** [comment]
I'm open to taking it off, but to bypass the event you have to individually open each machine on the network and boost its signal. It's the functional equivalent of replacing your APC's battery during the station blackout event. Sure you can do it, but most people just wait it out to avoid the work and usually there won't be someone with the proper access to fix it anyway.

It's just annoying as an engineer to (very frequently) tell the crew: "Sorry, I successfully figured out what was wrong, there's nothing to do about it but wait".

**Comment by PJB3005** [comment]
I made a complete rework of all the computers in telecomms once and then proceeded to burn out,

Then I handed it to Damian who also burned out.

https://github.com/PJB3005/vgstation13-1/tree/tcomms?files=1 if anybody cares.

**Comment by SonixApache** [comment]
wouldn't you only have to boost two or three machines to get it working again?

**Comment by Kurfursten** [comment]
Without going into the chamber and changing the linkups, the minimum number of machines you'd need would be 6 to get just common back, 7 if you want engineering too (it shares bus 4 with common). You can't just leave the hub disabled, for instance, even though you can build a working comm system without a hub. Similarly even though the server is only used for applying scripts and recording the chatlog, if the system is configured to go through it, it MUST be working or else nothing works because the signal never jumps back to the hub. If you want sec and command (share bus 3), add 4 more.

File: code/game/machinery/telecomms/telemonitor.dm [hunk 1]
```diff
@@ -97,6 +99,12 @@
 			if("mainmenu")
 				screen = 0
 
+			if("boost")
+				if(SelectedMachine.boost_signal())
+					temp = "<b>- SUCESS: \[[SelectedMachine]\] SIGNAL AMPLIFIED -</b>"
```
Comment (DamianX) [inline code/game/machinery/telecomms/telemonitor.dm]: SUCESS

**Comment by PJB3005** [comment]
@Kurfursten fix the typo, please.

**Comment by Kurfursten** [comment]
Fixed it here on github, thank you for web editor 🎆 

I'm literally 85% deep into a major port effort so I don't want to check out another branch atm

**Pull request merged**